src/addremove.js

```diff
@@ -0,0 +1,84 @@
+import { setToLocalStorage, getFromLocalStorage, reloadToDo } from './storage.js';
+import { refreshStore } from './functions';
+
+const editToDo = () => {
+  const toDoList = document.getElementsByClassName('task');
+  for (let i = 0; i < toDoList.length; i += 1) {
+    const taskLabel = toDoList[i].children[0].children[1];
+    taskLabel.addEventListener('click', () => {
+      refreshStore();
+    });
+  }
```

Comment:
- [ ] When I edit any task, it's not storing in the localstorage. Hence after refreshing the page, the new changes are not appearing. Please fix this issue.

Reply:
- [ ] Hi @Dipeshtwis, I am afraid this issue is not resolved yet. When I click on an item description to edit it automatically todo list object undefined. Kindly fix this bug and make sure it works as intended before you submit it for another review. 

![Screenshot from 2021-11-13 10-42-35](https://user-images.githubusercontent.com/64894927/141610580-37def225-4978-4c42-a860-0fb017637041.png)